manifests/profile/aws/filesystem.pp

```diff
@@ -10,24 +10,26 @@
 #   include nebula::profile::aws::filesystem
 class nebula::profile::aws::filesystem {
 
-  if $facts['disks']['xvdb'] {
-    filesystem { '/dev/xvdb':
-      ensure  => present,
-      fs_type => 'ext4',
-    }
+  if $facts['disks'] {
+    if $facts['disks']['xvdb'] {
```

Comment:
Was getting an error that `undef` doesn't respond to `[]`. Checking that `$facts['disks']` is defined fixes it, but there may be a cleaner fix.

Reply:
`if $facts['disks'] and $facts['disks']['xvdb']`?

Alternatively, we could set a default `$facts['disks']` fact - we should be able to rely on that being present everywhere, and it's just an artifact of the tests that it isn't.

Reply:
Done.